plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouter.sol

```diff
@@ -35,6 +36,7 @@ contract PaymentInFlightExitRouter is
     using PaymentDeleteInFlightExit for PaymentDeleteInFlightExit.Controller;
     using PaymentProcessInFlightExit for PaymentProcessInFlightExit.Controller;
     using BondSize for BondSize.Params;
+    using ExitBounty for ExitBounty.Params;
 
     // Initial IFE bond size = 185000 (gas cost of challenge) * 20 gwei (current fast gas price) * 10 (safety margin)
     uint128 public constant INITIAL_IFE_BOND_SIZE = 37000000000000000 wei;
```

Comment:
Should we update the INITIAL_IFE_EXIT_BOUNTY_SIZE?

Reply:
Not sure what you mean @thec00n ?
Do you mean update it to use 80 gwei as well?